pom.xml

```diff
@@ -3300,6 +3300,15 @@
       </modules>
     </profile>
 
+    <profile>
+      <id>no-shaded-client</id>
```

Comment:
The name looks too general. There are many shaded clients in Spark. Chill/Guava/etc.

Reply:
How about `no-shaded-hadoop-client`?

Reply:
+1